app/src/test/java/com/fstyle/structure_android/data/source/UserRepositoryTest.java

```diff
@@ -0,0 +1,104 @@
+package com.fstyle.structure_android.data.source;
+
+import com.fstyle.structure_android.data.model.User;
+import com.fstyle.structure_android.data.model.UsersList;
+import com.fstyle.structure_android.data.source.remote.UserRemoteDataSource;
+import com.fstyle.structure_android.data.source.remote.api.service.NameApi;
+import java.util.ArrayList;
+import java.util.List;
+import okhttp3.MediaType;
+import okhttp3.ResponseBody;
+import org.junit.After;
```

Comment:
ERROR: Wrong order for org.junit.After import. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports